src/cljs/ataru/hakija/application_form_components.cljs

```diff
@@ -143,10 +144,14 @@
         languages   (subscribe [:application/default-languages])
         size        (get-in field-descriptor [:params :size])
         size-class  (text-field-size->class size)
+        verify-email? (subscribe [:application/verify-email? id])
```

Comment:
Mun mielestä on ollut tapana tasata let-bindingsien oikea puolisko... Näitä on tässä muutama, mm. tossa `textual-field-change`ssa.